pip/qsharp/_qsharp.py

```diff
@@ -85,13 +123,13 @@ def callback(output: Output) -> None:
 def eval_file(path: str) -> Any:
```

Comment:
I think it's worth removing this now, as discussed offline.

Reply:
OK! Will do in separate PR since I'm rushing to merge this now.